developer/pages-router/localization.mdx

```diff
@@ -6,105 +6,114 @@ description: "This guide walks you through setting up localization for your Make
 </Note>
 
-<Frame>![Localization first step](/images/localization-first-step.png)</Frame>
-
-To add a new locale, click the "+ Add locale" button. You can modify or delete existing locales by hovering over the locale:
-
 <Frame>
-  ![Hovering over the locale](/images/localization-hover-locale.png)
+  ![Makeswift dashboard showing the root settings page for
+  Locales](/images/developer/guides/localization/localization-settings.jpeg)
 </Frame>
 
-You can also modify the default locale by hovering over the default locale and clicking the edit button.
-
-Once you add all the locales you need, it might look like this:
+To add a new locale, click the **+ Add locale** button. In this case, we've added **es-ES**. You can then modify and delete a locale (except the default locale) by hovering over it and clicking the settings icon.
 
 <Frame>
-  ![Manage locales in settings](/images/localization-manage-locales.png)
+  <img
+    src="/images/developer/guides/localization/secondary-locale.jpeg"
+    alt="Localization settings showing a secondary locale"
+  />
 </Frame>
 
-</Step>
+## Configure locales in your Next.js config
 
-<Step title="Configure locales in your Next.js config">
+The locales in your site settings in Makeswift need to match the locales configured in `next.config.ts`. Without matching these exactly, pages in the Visual Builder will not load properly.
 
-The locales in your site settings need to match the locales configured in `next.config.mjs`. For example, to match the locales on the screenshot above, update `next.config.mjs` to be like this:
+To match the locales on the screenshot above, update `next.config.ts` to be like this:
 
 <NextConfigLocalizedExample />
 
-Note that in `next.config.mjs`, you also need to put your `defaultLocale` in `locales`.
-For example, here, we're adding `en-US` to the `locales`.
-
-It is important to match the locales and the default locale on the site settings to the locales configured in `next.config.mjs`. Otherwise
-the pages on the builder will not load properly.
+Note that in `next.config.ts`, you also need to put your `defaultLocale` in `locales`.
+In this example, that would be `en-US`.
 
-</Step>
+## Fetch snapshots by locale
 
-<Step title="Update snapshot fetching">
-Once you set up locales in the site settings and in the Next.js config, you need to pass the locale from `getStaticProps` to `getPageSnapshot`.
+Update your catch-all route to fetch the page snapshots by locale. This requires you to pass the locale from `getStaticProps` to `getPageSnapshot`. This code is usually located in `[[...path]].tsx`, but it might be different depending on your setup.
 
 <PagesCatchAllLocalizedExample />
 
-This code is usually located in `[[...path]].tsx`, but it might be different depending on your setup.
+You'll also need to pass the locale to the `<ReactRuntimeProvider>` component by updating your `_app.tsx` like so.
 
-</Step>
+<PagesAppLocalized />
```

Comment:
I'd add a section/paragraph here along the lines of "you've successfully added a new locale and it's now live; you can now start adding localized content."

Reply:
Updated

Reply:
@jamesqquick Did you push your changes re: this one?

Reply:
In the next line. `After completing this setup, you can now start to add localized content in the Visual Builder.`

Reply:
I think it's important to underline that at this point the new locale is live. Maybe something like this?

At this point, the new locale is live, and site visitors can access the corresponding localized routes. However, until you localize your pages, they will continue to display content from the default locale. Read on to learn how to start localizing page content in the Visual Builder.